doc/softphone.md

```diff
@@ -176,6 +176,61 @@ arrive in a guaranteed order.
 - If you wish to control the MediaStream settings (i.e., input device) you can
 provide it as an option to `acceptSession` or as a default in the sdk's constructor.
 
+## V7 and Line Appearance
+v7 of the SDK introduces functionality to handle different Line Appearance values on the station.
+As such, this only applies to softphone sessions.
+
+This is a performance enhancement for the server and helps streamline the use of Persistent Connection
+within the SDK. Eventually, all stations will be migrated over to Line Appearance == 1
+(currently the default is 100). Clients should implement the SDK v7 to ensure the migration
+goes smoothly and no interruptions are experienced.
+
+### What Is Line Appearance?
+Line Appearance  is how many concurrent active webrtc sessions a user is allowed to have at a given time.
+Line Appearance will usually be **1** or **100**.
+For example:
+* If Line Appearance > 1 (ie. 100)
+  * Each webrtc phone call will receive its own Webrtc Session. Meaning if there are
+  3 active webrtc phone calls, there will be 3 corresponding Webrtc session.
+* If Line Appearancee == 1
+  * Each webrtc phone call will share the active Webrtc Session. Meaning if there is
```

Comment:
I think we should add a bit here so people really understand what is happening. What about something like:
```
All conversations will be multiplexed on the same session reducing local resource use as well as
improving call connection times. Only the audio from the active conversation will be heard and sent.
For example, if you have an active call and you answer or place another call, the original call will be
put on hold and the new call will become the active conversation. 
```

Reply:
👍🏼